corehq/apps/app_manager/remote_app.py

```diff
@@ -81,9 +81,6 @@ def make_remote_profile(app):
                     download_index_url=download_index_url
                 )
 
-        if app.build_langs:
-            profile_xml.set_property("cur_locale", app.build_langs[0])
-
```

Comment:
@sheelio or @amsagoff do you have opinions on whether we need language profiles to work for remote apps (logically it all still applies just the same, but I could see us punting if we're sure we don't want to support it). @theism do we need to continue to support the current Deploy langs behavior for Bihar (remote app)—the feature where you can choose to only include specific languages in the build?